common/python/ax/cluster_management/argo_cluster_manager.py

```diff
@@ -102,6 +115,54 @@ def uninstall(self, args):
         self._ensure_customer_id(uninstall_config.cloud_profile)
         ClusterUninstaller(uninstall_config).start()
 
+    def install_platform_only(self, args):
+        os.environ["AX_CUSTOMER_ID"] = "user-customer-id"
+        os.environ["ARGO_LOG_BUCKET_NAME"] = args.cluster_bucket
+        os.environ["ARGO_DATA_BUCKET_NAME"] = args.cluster_bucket
+        os.environ["ARGO_KUBE_CONFIG_PATH"] = args.kubeconfig
+
+        logger.info("Using customer id: %s", os.environ["AX_CUSTOMER_ID"])
+
+        args.silent = True
+        # 1. Create the platform install config first.
+        platform_install_config = PlatformOnlyInstallConfig(cfg=args)
+
+        config.load_kube_config(config_file=args.kubeconfig)
+        v1 = client.CoreV1Api()
+        ret = v1.list_node(watch=False)
+        assert len(ret.items) > 0, "There should at least be 1 node in the Kubernetes cluster"
```

Comment:
Validation of the provided cluster will have many different checks; one of them being the Kubernetes version. No checks done here or otherwise can guarantee that our pods will run on a user provided cluster.

Neways, I agree that the utility of this check is minimal at best. Removed.